config/settings.py

```diff
@@ -48,7 +48,9 @@
     "collectors.framework",
     "collectors.jiraffe",
     "collectors.nvd",
+    "collectors.osv",
     "collectors.product_definitions",
+    "collectors.ps_constants",
```

Comment:
does it make sense to have it if nobody is using it? what was the rationale for this collector?

Reply:
While I was checking snippets in OCP pre-prod, I found out that there is nothing from `osv` because `osv` was not in `settings.py`. I noticed that `ps_constants` was also missing,  so I asked @osoukup whether it should be there, and he said yes (but I do not know how `ps_constants` exactly works).

Reply:
PS constants are definitely needed but they are needed by the tracker filing functionality so nobody could realize the missing data yet. This is my mistake not to set the app to start the periodic sync. I probably did the sync one time locally when developing it and then did not realize that the data is not being updated.